scripts/teamDataScript.js

```diff
@@ -0,0 +1,617 @@
+        var preMessageSpace = isSeasonSaved() ? '' : '<br><br>';
+    
+        modalContentHTML += preMessageSpace + '<p>No event is currently checked. Go to the <a href="event-data.html">Event Data</a> page '+ 
+        'to enter new events and teams attending those events for a selected season.' + 
+        '<br><br>Make sure you check the new event you enter.</p>';
+    }
+    
+    $('#tDataModalContent').append(modalContentHTML);
+    $('#teamDataAlertModal').show();
+}
+
+function closeTDataModal() //onclick for x button in modal
+{
+    $('#teamDataAlertModal').hide();
+}
+
+function displayTeamRadioList()
+{
+    var season = getCookie('currCheckedSeason');
+    var event = getCurrEvent();
+    if (season.trim().length == 0 || event.trim().length == 0) return;
+
+    var teamsJSON = getTeams(season,event);
+    var keys = Object.keys(teamsJSON);
+    for (var i = 0; i < keys.length; i++)
+    {
+        var teamNumber = keys[i];
+        var teamName = teamsJSON[teamNumber];
+
+        var newItem = "<label><input type=\"radio\" name=\"teamListItem\" value=\"" + teamNumber + "\">" + `${teamNumber}: ${teamName}` + "</label><br>";
+        $("#teamRadioList").append(newItem); //add to radiolist
+    }
+}
+
+//onchange for the team radio list
+function changeCurrTeam()
+{
+    var currTeamNumber = $("input[name=teamListItem]:checked","#teamRadioList").val(); //in prev method, val of each radio item is the tNum
+    var currSeason = getCookie('currCheckedSeason');
+    var currEvent = getCurrEvent();
+    var cname = 'currCheckedTeam';
+    var currTeamName = getTeams(currSeason,currEvent)[currTeamNumber];
+
+    if (currSeason.trim().length == 0 || currEvent.trim().length==0 || currTeamNumber.trim().length == 0)
+    {
+        $("#teamStatsTitle").text('');
+        return;
+    }
+
+    var cvalue = `/${currSeason}/${currEvent}/${currTeamNumber}`;
+    setCookie(cname,cvalue,750);     //curr team cookie --> currCheckedTeam = /{season}/{event}/{currTeamNumber}
+
+    $("#teamStatsTitle").text(`Data for team ${currTeamNumber} : ${currTeamName}`);
+    $('#pitScoutingTitle').text(`Pit Scouting Questions and Answers for team ${currTeamNumber}: ${currTeamName}`);
+    $('.js_clear_on_load').val("").html("");
+
+    loadPit();
+    loadMatch();
+    localStorage.setItem('currCheckedMatch','');
+}
+
+//basically copy pasted from w3schools, onclick for a change in tabs
+function displayTabContent(evt, tabName)
+{
+    var i, x, tablinks;
+    x = document.getElementsByClassName("tabContent");
+    for (i = 0; i < x.length; i++) {
+      x[i].style.display = "none";
+    }
+    tablinks = document.getElementsByClassName("tabs");
+    for (i = 0; i < x.length; i++) {
+      tablinks[i].className = tablinks[i].className.replace(" w3-bottombar-gold", " w3-bottombar-maroon");
+    }
+    document.getElementById(tabName).style.display = "block";
+    evt.currentTarget.firstElementChild.className += " w3-bottombar-gold";
```

Comment:
little bit of jQuery can shorten this up:

```js
   x = document.getElementsByClassName("tabContent");
    for (i = 0; i < x.length; i++) {
      x[i].style.display = "none";
    }
```
becomes:
```js
$('.tabContent').hide();
```

and 

```js
    tablinks = document.getElementsByClassName("tabs");
    for (i = 0; i < x.length; i++) {
      tablinks[i].className = tablinks[i].className.replace(" w3-bottombar-gold", " w3-bottombar-maroon");
    }
```

becomes
```js
$('.tabs').removeClass('w3-bottombar-gold').addClass('w3-bottombar-marron');
```

and lastly,
```js
document.getElementById(tabName).style.display = "block";
```

becomes
```js
$('#' + tabName).show();
```

(just a thought :smile:  )